### ISSUE TITLE
consolidate front-end framework

### ISSUE THREAD

**Issue by ckmah** [shassathe/covid_qPCR_test#34]
just realized templates are currently mixing `bootstrap` + `tailwind` which may break some mobile responsive elements. i will convert to one

**Comment by jwill490** [comment]
Yeah I was worried about this as well, but I just wanted to get the UI up ASAP. I don't like using bootstrap because it looks v generic. I'm pretty sure all we need to do is (a) replace the crispy form fields with tailwind-formatted form fields, and (b) replace the jumbotrons with tailwind headers

**Comment by jwill490** [comment]
Alternatively you could leave the bootstrap and convert the tailwind classes to plain CSS, but replacing bootstrap is probably easier.

**Comment by ckmah** [comment]
yeah i think i will convert to bootstrap only bc i'm more familiar with it, would of preferred to learn tailwind if we had more time though!
